packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -11,12 +12,19 @@ jest.mock('@salesforce/core', () => {
 });
 
 describe('test project paths', () => {
+  const hasRootWorkspaceStub = jest.spyOn(workspaceUtils, 'hasRootWorkspace');
+  const FAKE_HOME_DIR = '/fake/home/dir/';
   const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
   const FAKE_STATE_FOLDER = '.sfdx';
+  const FAKE_CONFIG_FILE = 'sfdx-config.json';
+  const FAKE_PATH = path.join(FAKE_WORKSPACE, FAKE_STATE_FOLDER);
+  let homeDirStub: jest.SpyInstance;
 
+  beforeEach(() => {
+    homeDirStub = jest.spyOn(os, 'homedir').mockReturnValue(FAKE_HOME_DIR);
+  });
```

Comment:
I think this beforeEach can be removed now also since the global function was removed.